pkg/app/middlewares/server/recovery/recovery.go

```diff
@@ -37,15 +35,16 @@ var (
 )
 
 // Recovery returns a middleware that recovers from any panic and writes a 500 if there was one.
-func Recovery() app.HandlerFunc {
+// By overriding the Config configuration, you can customize the error printing logic.
```

Comment:
I think the comments here need to describe the default behavior.

Reply:
i got it, thx